lib/tasks/accounts.rake

```diff
@@ -0,0 +1,74 @@
+namespace :accounts do
+  desc "merge two user accounts using environment variables SRC=<username-1> DST=<username-2>"
+  task merge: [:connection] do
+    if !ENV['SRC'] || !ENV['DST']
+      STDERR.puts "Usage: SRC=<username-1> DST=<username-2> rake accounts:merge"
+      exit 1
+    end
+
+    require 'exercism/user'
+
+    src = User.where(username: ENV['SRC']).first
+    if !src
+      STDERR.puts "Cannot find user record for %s" % ENV['SRC']
+      exit 1
+    end
+
+    dst = User.where(username: ENV['DST']).first
+    if !dst
+      STDERR.puts "Cannot find user record for %s" % ENV['DST']
+      exit 1
+    end
+
+    require 'exercism/user_exercise'
+    require 'exercism/named'
+    require 'exercism/problem'
+    require 'exercism/submission'
+    require 'exercism/acl'
+
+    src.exercises.each do |src_exercise|
+      # sometimes people have done the same exercise on both accounts
+      target = UserExercise.where(language: src_exercise.language, slug: src_exercise.slug, user_id: dst.id).first
+      if !!target
```

Comment:
Typically there will not be a `target`.

> it is especially important somehow that this conditional must be a true literally

I think this is where you and I have very different experiences in Ruby. To me this is the idiom. There's nothing special about it, and it's not intended to say "heads up warning this could be something surprising". It's just a boolean, explicitly.